File: Homeworks/OperatorsAndExpressions/PointInACircle/PointInACircle.cs
using System;

//Problem 7. Point in a Circle

//Write an expression that checks if given point (x, y) is inside a circle K({0, 0}, 2).

class PointInACircle
{
    static void Main()
    {
        Console.WriteLine("Please enter coordinate x: ");
        string xCoordinate = Console.ReadLine();
        Console.WriteLine("Please enter coordinate y: ");
        string yCoordinate = Console.ReadLine();

        const int radius = 2;

        bool isInCircle = Math.Pow(double.Parse(xCoordinate), 2) + Math.Pow(double.Parse(yCoordinate), 2) <=
                          Math.Pow(radius, 2);

        if (isInCircle)
        {
            Console.WriteLine("The point is in the circle!");
        }
        else
        {
            Console.WriteLine("The point is not in a circle!");
        }
    }
}

File: Homeworks/Javascript-Basics/OperatorsAndExpressions/scripts/pointInACircle.js
taskName = "Point in a circle";

function Main(bufferElement) {

    var xCoordinate = ReadLine("Please enter x coordinate of the point: ");
    var yCoordinate = ReadLine("Please enter y coordinate of the point: ");
    var xCoordCircle = 0;
    var yCoordCircle = 0;
    var radius = 5;

    SetSolveButton(function () {
        var isInCircle = IsInCircle(xCoordinate.value, yCoordinate.value, xCoordCircle, yCoordCircle, radius);

        WriteLine(isInCircle);
    });
}

function IsInCircle(xCoord, yCoord, xCoordCircle, yCoordCircle, radius) {

    if(Math.pow(xCoord - xCoordCircle, 2) + Math.pow(yCoord - yCoordCircle, 2) < Math.pow(radius, 2)) {
        return true;
    }
    else {
        return false;
    }
}

